### ISSUE TITLE
bdsp in optimized showdown fork

### ISSUE THREAD

**Issue by mancho2000** [hsahovic/poke-env#245]
Hey Haris,

Any chance we can get the new bdsp modes in your optimized showdown fork? :)

**Comment by hsahovic** [comment]
Hey @mancho2000,

The optimized fork is not maintained anymore, as all its features have been merged into the main showdown repo. You just need to do `node pokemon-showdown start --no-security` on an up-to-date version of the official server, and you'll get all the benefits of the optimized fork in terms of throttling, ease of use and performance, while still being up-to-date in terms of data and formats.

**Comment by mancho2000** [comment]
Hey,

Thanks, I did this but now my agent is not starting any battles. I tried manually starting a random battle against myself to check that the server is working properly and it is. 
This is the message I get:

2022-01-13 09:56:55,066 - SimpleRLPlayer 1 - WARNING - Received pm: |pm| SimpleRLPlayer 1| MixedDamagePlaye 1|/challenge gen8randombattle|gen8randombattle|||
2022-01-13 09:56:55,066 - MixedDamagePlaye 1 - WARNING - Received pm: |pm| SimpleRLPlayer 1| MixedDamagePlaye 1|/challenge gen8randombattle|gen8randombattle|||
2022-01-13 09:56:55,067 - MixedDamagePlaye 1 - WARNING - Received pm: |pm| SimpleRLPlayer 1| MixedDamagePlaye 1|/log SimpleRLPlayer 1 wants to battle!
2022-01-13 09:56:55,068 - SimpleRLPlayer 1 - WARNING - Received pm: |pm| SimpleRLPlayer 1| MixedDamagePlaye 1|/log SimpleRLPlayer 1 wants to battle!

It works normally with the previous server that I was using, what could be the problem?
